library/agent/hooks/wrapRequire.ts

```diff
@@ -206,19 +206,24 @@ function patchPackage(this: mod, id: string, originalExports: unknown) {
     );
   }
 
+  const agent = getInstance();
+  agent?.onPackageRequired(moduleName, installedPkgVersion);
+
+  // We don't want to patch this package because we do not have any hooks for it
+  if (!versionedPackages.length) {
```

Comment:
should we rename?

Reply:
https://github.com/AikidoSec/firewall-node/pull/454/commits/8560275f272ca67e625904a646eb051541ad8fcc